ironic-scripts/runmariadb

```diff
@@ -18,6 +19,13 @@ fi
 
 ln -sf /proc/self/fd/1 /var/log/mariadb/mariadb.log
 
+# Restart mysqld when the certificate is updated
+if [ -f "$MARIADB_CERT_FILE" && "${RESTART_CONTAINER_CERTIFICATE_UPDATED}" == "true" ]; then
+    inotifywait -m -e delete_self "${MARIADB_CERT_FILE}" | while read file event; do
+       kill $(pgrep mysqld)
```

Comment:
`mysqld_safe` automatically restarts if mysqld exits, so this is consistent with the commit message but inconsistent with the implementation of the other services (where we exit the container).

Reply:
Thanks! Fixed!